cardano-node/src/Cardano/Tracing/OrphanInstances/Network.hs

```diff
@@ -1770,7 +1769,7 @@ instance (Show versionNumber, ToJSON versionNumber, ToJSON agreedOptions)
       [ "kind" .= String "HandshakeServerError"
       , "reason" .= toJSON err
       ]
-  toObject _verb (TrError e err cerr) =
+  toObject _verb (TrConnectionHandlerError e err cerr) =
     mconcat
       [ "kind" .= String "Error"
```

Comment:
In PR #4230 this was serialized as `"kind" .= String "ConnectionHandlerError"`. Did you decide to keep the previous serialization / is that change obsolete?

Reply:
Thanks for the pointing this out.  Fixed now.